### ISSUE TITLE
Fields marked as sensitive is included in emails

### ISSUE THREAD

**Issue by mastrup** [umbraco/Umbraco.Forms.Issues#52]
When marking fields as sensitive, the fields is still included the email that is sent.

In my specific case, I need to exclude a CPR (social security number) from being sent - The value of the field is not allowed to leave Umbraco.
I would expect that fields marked as sensitive ([https://i.imgur.com/0y15WI4.jpg](https://i.imgur.com/0y15WI4.jpg)) isn't included in any email that is sent from Forms or at least I have an option to exclude them.

A solution to this could be to extend the FormFieldHtmlModel with a boolean value that indicates whether the field is sensitive or not. In the email template it could be handled like this:

```
@foreach (var field in Model.Fields.Where(x => !x.IsSensitive))
{
    ...
}
```

**Comment by nul800sebastiaan** [comment]
Thanks @mastrup - I can reproduce and there's currently not an easy way to exclude sensitive fields from the emails.

**Comment by greystate** [comment]
I was just going to report this but I see you beat me to it @mastrup - _by more than a year_ 😆 😿

**Comment by warrenbuckley** [comment]
Fixed in this PR - https://github.com/umbraco/Forms/pull/329

**Comment by warrenbuckley** [comment]
Has been re-opened due to implementation details & has a new PR for review - https://github.com/umbraco/Forms/pull/334

**Comment by warrenbuckley** [comment]
OK re-tested & merged in this PR @nul800sebastiaan 
https://github.com/umbraco/Forms/pull/334

**Comment by nul800sebastiaan** [comment]
Just to make clear what is happening here, we had a few chats about this at HQ: this could be considered a breaking change since we're no longer sending the sensitive data by default, so you'd need to opt in to sensitive data being sent... and the other way around, we don't want to keep the current behavior where sensitive data just ends up in emails. 

So here's what we've done:

- As of 8.4.0 - the 3 built-in workflows for sending emails will no longer send fields marked as sensitive by default, you can still opt-in to those fields being sent though
- Same goes for the Slack workflow - it does not send fields marked as sensitive any more until you opt back in
- For any other workflows it is difficult for us to make a decision, you might already be connecting the data to an external system in which it is safe to process and store this data. We did not want to break existing integrations by withholding data from fields marked as sensitive, so they will still be sent
- Any new workflows you define will have the "Include Sensitive Data" checkbox disabled by default so it's always a conscious opt-in to send this data into a workflow

In short: after upgrading, existing email and Slack workflows no longer send sensitive data, you need to manually opt in to that. All other existing workflows will keep sending sensitive data until you explicitly opt out for those workflows. 
New workflows have a checkbox that is disabled by default, allowing you to include sensitive data in the workflow (see below).

![image](https://user-images.githubusercontent.com/304656/76865862-9fc8a800-6863-11ea-864a-2f6ad305d374.png)
